SSToastMessage/Example/SSToastMessage/ContentView.swift

```diff
@@ -66,7 +66,9 @@ struct ContentView: View {
                     self.createAlertView()
                 }
                     
-                .present(isPresented: self.$showTopToast, type: .toast, position: .top, onTap: {print("on toast tap")}) {
+                .present(isPresented: self.$showTopToast, type: .toast, position: .top, autohideDuration: 6.0, onTap: {print("on toast tap")}, onToastHide: {
```

Comment:
@krupa-p-simformsolutions  the example instead of adding a print statement, can we perform a small UI change or new screen redirection? That would give more ideas to the end developer.